run_project_tests.py

```diff
@@ -1622,17 +1623,21 @@ def clear_transitive_files() -> None:
         (passing_tests, failing_tests, skipped_tests) = res
     except StopException:
         pass
-    print()
-    print('Total passed tests: ', green(str(passing_tests)))
-    print('Total failed tests: ', red(str(failing_tests)))
-    print('Total skipped tests:', yellow(str(skipped_tests)))
     if failing_tests > 0:
         print('\nMesonlogs of failing tests\n')
```

Comment:
I see you kep this line, but I don't see it in the logs you pasted. Shouldn't this be deleted?

Reply:
I do see it though:
```
 [SKIPPED]   wayland: 2 core only

Total configuration time: 1877.07s
Total build time:         854.80s
Total test time:          76.47s

Mesonlogs of failing tests


=========================================== common: 2 cpp ==========================================
 

(inprocess) $ setup --prefix /usr --libdir lib 'test cases/common/2 cpp' '/cygdrive/d/a/meson/meson/b 1f81195594' --backend=ninja
```

Reply:
Oh sorry, that's indeed before the failed tests logs.